server/planning/assignments/assignments_content.py

```diff
@@ -94,7 +94,12 @@ async def get_item_from_assignment(assignment, template=None):
 
                 await merge_subject(item, planning)
                 merge_list("place", item, planning)
-                merge_list("anpa_category", item, planning)
+
+                anpa_category = planning_data.get("anpa_category")
+                if anpa_category:
+                    item["anpa_category"] = deepcopy(anpa_category)
+                else:
+                    merge_list("anpa_category", item, planning)
```

Comment:
nit: you can take advantage of the walrus operator here 😁 
```suggestion
                if anpa_category := planning_data.get("anpa_category"):
                    item["anpa_category"] = deepcopy(anpa_category)
                else:
                    merge_list("anpa_category", item, planning)
```